extracters/knoedler_data.py

```diff
@@ -233,7 +233,8 @@ def make_objects_names(data, gpi=None, uuid_cache=None):
 
 @use('gpi')
 @use('uuid_cache')
-def make_objects_tags_ids(data, gpi=None, uuid_cache=None):
```

Comment:
Yep. Sorry I missed these when I split off the aat database.

Reply:
No worries :)